terraform/environments/production/gateways.tf

```diff
@@ -1,7 +1,27 @@
 # Deploy our dogfood gateways
 locals {
-  gateways_region = local.region
-  gateways_zones  = [local.availability_zone]
+  gateways_region         = local.region
+  gateways_zone           = local.availability_zone
+  gateways_instance_type  = "e2-highcpu-2"
```

Comment:
This is cheaper we can go that can be provisioned and doesn't violate our quotas.